src/templates/COLORED_FOLDER.eta

```diff
@@ -0,0 +1,285 @@
+/*
+Generated by Snippetor on <%= it.date %>.
+This file will be overwritten if you push the button again...
+*/
+<%
+const root = it.cfg.name;
+
+function getForeground(e, light) {
+    value = light ? e.lightMode.foreground : e.darkMode.foreground;
+    return value ? value : '"inherit"'
+}
+function getBackground(e, light) {
+    value = light ? e.lightMode.background : e.darkMode.background;
+    return value ? value : '"transparent"'
+}
+
+function defineColors(i, folder) {
+%>
+/* for folder <%~ folder.target %> */
+.theme-light {    
+    /** Setup Custom Colors */
+    --<%~ root %>-folder-text-<%= i %>: <% tR += getForeground(folder, true) %>;
+    --<%~ root %>-folder-<%= i %>: <% tR += getBackground(folder, true) %>;
+    --<%~ root %>-folder-radius: 5px;
+}
+.theme-dark {    
+    /** Setup Custom Colors */
+    --<%~ root %>-folder-text-<%= i %>: <% tR += getForeground(folder, false) %>;
+    --<%~ root %>-folder-<%= i %>: <% tR += getBackground(folder, false) %>;
+    --<%~ root %>-folder-radius: 5px;
+}
+<%
+} // end defineColors
+function folderSettings(i, folder) {
+%>
+
+.nav-folder.mod-root {
+    --<%~ root %>-border-radius: <%= it.cfg.borderRadius %>px;
```

Comment:
Is this a remnant global (all folders) setting? (or an intentional one that needs a specific setting and should be guarded/moved? (this is within the folderSettings method, but seems to define/reference a broader config element...